### ISSUE TITLE
SPIFFS PANIC'ED event called  (IDFGH-551)

### ISSUE THREAD

**Issue by filzek** [espressif/esp-idf#2934]
The SPIFFS doesnt work inside a secodary task, if SPIFFS are called from a task it deploy a guru panic'ed event. Tried everything to solve but was same result.

Can use the spiffs example and just move inside a task within a task and it will fire a exception.

**Comment by igrr** [comment]
Hi @filzek, when you were opening this issue, you were presented with the issue template. Could you please supply information requested therein? In particular, IDF version, crash logs (preferrably decoded using IDF Monitor), code to reproduce the issue. Thanks.

**Comment by filzek** [comment]
We found out the problem run on SPIFFS and in FAT, but in FAT if we increase the task heap it works, in SPIFFS never works same problem. We decided to change to FAT as it is working ok.

Default settings global.
Create a Initial Task that call other tasks from it.
After the 2nd task call the thrid task to mount the spiffs system hang with the SPIFFS PANIC event.

Same occurs with FAT, increasing the task heap solve the problem in FAT, but doesnt solve the problem in SPIFFS.
